specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/dscCompilationJob.json

```diff
@@ -107,6 +107,11 @@
         "externalDocs": {
           "url": "http://aka.ms/azureautomationsdk/dsccompilationjoboperations"
         },
+        "x-ms-examples": {
+          "Get a DSC Compilation job": {
+            "$ref": "./examples/getCompilationJob.json"
```

Comment:
Validation Failure: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/338705368#L755

Reply:
Hey, I checked with the team and I resolved the issue. They had added new flags and it wasn't documented in the spec, which caused the validation failures in DSCCompilationJob. Fixed it.